modules/nf-core/gsea/gsea/main.nf

```diff
@@ -4,51 +4,48 @@ process GSEA_GSEA {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/gsea:4.3.2--hdfd78af_0':
-        'biocontainers/gsea:4.3.2--hdfd78af_0' }"
+        'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/0f/0f4fe28961396eeeaa98484cb4f2db5c79abfdf117700df132312fe5c41bff81/data':
+        'community.wave.seqera.io/library/gsea:4.3.2--a7421d7504fd7c81' }"
 
     input:
     tuple val(meta), path(gct), path(cls), path(gene_sets)
     tuple val(reference), val(target)
     path(chip) // Optional identifier mapping file
 
     output:
-    tuple val(meta), path("*.rpt")                             , emit: rpt
-    tuple val(meta), path("*index.html")                       , emit: index_html
-    tuple val(meta), path("*heat_map_corr_plot.html")          , emit: heat_map_corr_plot
-    tuple val(meta), path("*gsea_report_for_${reference}.tsv") , emit: report_tsvs_ref
-    tuple val(meta), path("*gsea_report_for_${reference}.html"), emit: report_htmls_ref
-    tuple val(meta), path("*gsea_report_for_${target}.tsv")    , emit: report_tsvs_target
-    tuple val(meta), path("*gsea_report_for_${target}.html")   , emit: report_htmls_target
-    tuple val(meta), path("*ranked_gene_list*.tsv")            , emit: ranked_gene_list
-    tuple val(meta), path("*gene_set_sizes.tsv")               , emit: gene_set_sizes
-    tuple val(meta), path("*global_es_histogram.png")          , emit: histogram
-    tuple val(meta), path("*heat_map_1.png")                   , emit: heatmap
-    tuple val(meta), path("*pvalues_vs_nes_plot.png")          , emit: pvalues_vs_nes_plot
-    tuple val(meta), path("*ranked_list_corr_2.png")           , emit: ranked_list_corr
-    tuple val(meta), path("*butterfly_plot.png")               , emit: butterfly_plot, optional: true
-    tuple val(meta), path("*[!gene_set_size|gsea_report|ranked_gene_list]*.tsv"), emit: gene_set_tsv, optional: true
-    tuple val(meta), path("*[!gsea_report|heat_map_corr_plot|index|pos_snapshot|neg_snapshot]*.html"), emit: gene_set_html, optional: true
+    tuple val(meta), path("*.rpt")                                                                                                       , emit: rpt
```

Comment:
Ok, I'm going to say that this amount of whitespace is ridiculous, and we should just align the top emits and then leave the bottom two.